lib/medium-admin.php

```diff
@@ -765,12 +765,9 @@ public static function cross_post($post, $medium_post, $medium_user) {
     $tags = array_values(array_unique(array_merge($slugs, $tags)));
 
     // pull in categories as tags
-    $categories = wp_get_the_category($post->ID);
-    $cats = array();
-    foreach($categories as $category){
-        $cats[] = $category->name;
-    }
-    $tags = array_values(array_unique(array_merge($cats, $tags)));
+    $args = array('fields' => 'names'); // @see wp_get_post_categories() $args ref to wp_get_object_terms()
```

Comment:
double quotes, as the rest of this plugin please, even though it's not WP-standard.